src/components/SearchForm/SearchForm.js

```diff
@@ -35,16 +35,16 @@ function SearchForm({ isLoggedIn }) {
   };
 
   return (
-    <div className="Search-Form">
-      <div className="Search-Form__search-bar">
+    <section className="search-form"> 
+      <div className="search-form__search-bar">
         <input
```

Comment:
If you have `inputs` then they should be inside tag `form` to correctly handle form submitting